lib/vsc/filesystem/operator.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: latin-1 -*-
+#
+# Copyright 2009-2022 Ghent University
+#
+# This file is part of vsc-filesystems,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-filesystems
+#
+# All rights reserved.
+#
+"""
+Interface to dynamically load vsc.filesystem modules and instantiate its XxxxOperations class
+
+@author: Alex Domingo (Vrije Universiteit Brussel)
+"""
+import logging
+
+def load_storage_operator(storage):
+    """
+    Load and initialize corresponding operator class for this filesystem
+    Return *Operations object
+
+    @type storage: Storage object from vsc.config
```

Comment:
stroage attribute from VscStorage instance (which is a Storage)

Reply:
Done in https://github.com/hpcugent/vsc-filesystems/pull/83/commits/b15712c7b826a3f3e53efbede1f68e450a12e987